crates/kitsune_p2p/dht_arc/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "kitsune_dht_arc"
+version = "0.0.1"
+authors = ["Michael Dougherty <maackle.d@gmail.com>"]
```

We should be in the habit of copying Cargo.toml files... to be releasable they need all these fields too:

```suggestion
description = "DhtArc types and logic"
license = "Apache-2.0"
homepage = "https://github.com/holochain/holochain"
documentation = "https://github.com/holochain/holochain"
authors = [ "Holochain Core Dev Team <devcore@holochain.org>" ]
keywords = [ "holochain", "holo", "p2p", "dht", "networking" ]
categories = [ "network-programming" ]
```